examples/multi-module/main.py

```diff
@@ -27,4 +27,4 @@ def done(self):
 # Start a main chare of type Main. We specify to the charm runtime which
 # modules contain Chare definitions. Note that the __main__ module is always
 # searched for chare definitions, so we don't have to specify it
-charm.start(Main, modules=['hello', 'goodbye'])
```

Comment:
Why not add another example that uses ```@register```, rather than removing this one? I think it's fine if they exist side-by-side.